src/trix/config/block_attributes.coffee

```diff
@@ -5,6 +5,10 @@ Trix.config.blockAttributes = attributes =
   quote:
     tagName: "blockquote"
     nestable: true
+  heading:
```

Comment:
I like the name "heading" (or "header") and think the name fits well the notion of Everyday Writing™. I can imagine "subheading" for `<h2>` and perhaps "author" for `<h3>` in a blogging app. Point being that I think adding distinct names is a good constraint. What are you going to do with `<h5>` and `<h6>` if you can't name it clearly? And people are of course free to configure the attribute names as they see fit.